srsenb/src/stack/upper/s1ap.cc

```diff
@@ -1115,7 +1115,7 @@ bool s1ap::ue::send_initial_ctxt_setup_response(const asn1::s1ap::init_context_s
     auto& item = container.erab_setup_list_ctxt_su_res.value[i].value.erab_setup_item_ctxt_su_res();
     item.transport_layer_address.resize(32);
     uint8_t addr[4];
-    inet_pton(AF_INET, s1ap_ptr->args.gtp_bind_addr.c_str(), addr);
+    inet_pton(AF_INET, s1ap_ptr->args.gtp_advertise_addr.c_str(), addr);
```

Comment:
i would propose to set the `gtp_advertise_addr` to `gtp_bind_addr` by default and only overwrite it when the user specifies it

Reply:
done